README.md

```diff
@@ -16,7 +16,7 @@ The Subnet EVM runs in a separate process from the main AvalancheGo process and
 ```
 [v0.1.0] AvalancheGo@v1.7.0-v1.7.4
 [v0.1.1-v0.1.2] AvalancheGo@v1.7.5-v1.7.6
-[v0.2.0] AvalancheGo@v1.7.7-v1.7.8
+[v0.2.0] AvalancheGo@v1.7.7-v1.7.9
```

Comment:
Should this be v1.7.10 in anticipation of the next AvalancheGo release tomorrow?

Reply:
I'll make that change when the release is ready